### DISCUSSION TITLE
cpu_accumulator.go中的takeCPUs

### DISCUSSION THREAD

**Discussion by dykhliww** [koordinator-sh/koordinator#1192] [General]
if acc.needs(acc.topology.CPUsPerCore())这块的代码有个break，只是跳出了当前循环，是否也应该跳出外层循环

**Comment by saintube** [comment]
/cc @eahydra 
代码问题欢迎提交 issue 反馈，在这里 discussion 部分大家不容易第一时间看到~

**Reply by eahydra** [reply]
> if acc.needs(acc.topology.CPUsPerCore())这块的代码有个break，只是跳出了当前循环，是否也应该跳出外层循环

@dykhliww  非常赞！确实是需要跳出外层循环的。有意愿提个PR优化下么？
